lib/formulator.ex

```diff
@@ -2,31 +2,80 @@ defmodule Formulator do
   use Phoenix.HTML
   alias Formulator.HtmlError
 
-  def input(form, field, options \\ []) do
+  @doc  """
+  Returns an html input with an associated label.
```

Comment:
In the case of errors, `input` will also render an error message after the input. <- that too, right?